docs/source/developer_documentation/boefjes-runner.md

```diff
@@ -236,11 +236,60 @@ https://developer.hashicorp.com/nomad/tutorials/manage-jobs/jobs-accessing-logs:
 
 [nomad-logs-api]: https://developer.hashicorp.com/nomad/api-docs/client#stream-logs
 
+## Building images with this spec from the current boefjes
+
+The approach to building OCI images from the boefjes we currently have in our
+system has been discussed in [this ticket][ticket], with the first versions
+having been implemented in these PRs:
+- https://github.com/minvws/nl-kat-coordination/pull/2709
+- https://github.com/minvws/nl-kat-coordination/pull/2832
+
+
+#### Summary of decisions
+We decided not to focus on the following:
+- We are **not** going to provide plain zip archives in the near future.
+- Discoverability of images from external repositories (potentially containing
+  multiple boefjes) will be pushed to later versions of OpenKAT.
+
+In terms of how we are going to build images, we decided to:
+- Just leverage Docker as this has to be available for OpenKAT devs anyway.
+- Aim to keep the build scripts flexible but simple, e.g. for `kat_dnssec` we have:
+```
+docker build -f ./boefjes/plugins/kat_dnssec/boefje.Dockerfile -t openkat/dns-sec --build-arg BOEFJE_PATH=./boefjes/plugins/kat_dnssec .
+```
+- Use, as shown above, the [naming convention][dockerfile-naming] for Dockerfiles
+  since we may want to add normaliser Dockerfiles in the same directory.
+- Use a Python base image for all our boefjes, so we can use shared Python code to
+  communicate with the boefjes API. Since there is no one tool available across Docker base images
```

Comment:
What do you mean with "no one tool available across Docker base images that can perform HTTP communication"

Reply:
There are *many* tools that a container might  use. I do think adding this upload functionality inside the 'thin' wrapper around the original  container, and ideally not rely on python being included. Curl of even wget works just as well I'd think.

Reply:
Docker images come in many sizes and formats. Alpine images do not have `curl` and images could even not have [a shell](https://stackoverflow.com/questions/59094873/can-we-have-a-docker-container-without-a-shell). Forcing different "native" scripts into the current set of boefje images is therefore probably more work to get right than take one extra step to install python and httpx. Later we can look into a binary per platform that does "just work".

Reply:
If we can install python we can install Curl, but for now this is fine. Lets just keep the docs as simple as possible and at least mention the possibility of using tools like wget, curl or another http client.